backend/app/transcendence/src/app.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get } from "@nestjs/common";
+import { AppService } from "./app.service";
+
+@Controller()
+export class AppController
+{
+	constructor(private readonly appService: AppService)
+	{
+
+	}
+
+	@Get()

```

Comment:
if I understand correctly, if we are willing to keep the habit of adding `;` at the end of our lines there should be one here, right?

Reply:
Yes we are want to have this habit, except, this is a decorator, identifiable by @. It take part as one statement @Get() getHello(): string.
Decorator must be in new line to gave understandable logic.
We can see in other file other subtility.

Reply:
Decorator override functionnality.